modules/storage/redis-cache/README.md

```diff
@@ -4,43 +4,39 @@ This module deploys Azure Cache for Redis(Microsoft.Cache/redis) and optionally
 
 ## Details
 
-[Azure Cache for Redis](https://azure.microsoft.com/en-us/pricing/details/cache/)  Azure Cache for Redis gives you the ability to use a secure open source Redis cache. It's a dedicated offering managed by Microsoft, to build highly scalable and responsive applications by providing you super-fast access to your data. You get to leverage the rich feature set and ecosystem provided by Redis
+{{ Add detailed information about the module. }}
```

Comment:
Can you restore the details section?

Reply:
BTW, is the Details section removed by `brm generate`? If yes, we may have a bug.

Reply:
fixed, there's no bug. it has been my fault yesterday.

Reply:
Oh got it. Thanks for confirming!